tests/aws/services/stepfunctions/v2/callback/test_callback.py

```diff
@@ -697,3 +698,69 @@ def test_sqs_wait_for_task_token_no_token_parameter(
             definition,
             exec_input,
         )
+
+    @markers.aws.validated
+    @pytest.mark.parametrize(
+        "template",
+        [CT.SQS_PARALLEL_WAIT_FOR_TASK_TOKEN, CT.SQS_WAIT_FOR_TASK_TOKEN_CATCH],
+        ids=["SQS_PARALLEL_WAIT_FOR_TASK_TOKEN", "SQS_WAIT_FOR_TASK_TOKEN_CATCH"],
+    )
+    def test_sqs_failure_in_wait_for_task_tok_no_error_field(
+        self,
+        aws_client,
+        create_iam_role_for_sfn,
+        create_state_machine,
+        sqs_create_queue,
+        sfn_snapshot,
+        template,
+        request,
+    ):
+        if (
+            request.node.name
+            == "test_sqs_failure_in_wait_for_task_tok_no_error_field[SQS_PARALLEL_WAIT_FOR_TASK_TOKEN]"
+        ):
+            # TODO: The conditions in which TaskStateAborted error events are logged requires further investigations.
+            #  These appear to be logged for Task state workers but only within Parallel states. The behaviour with
+            #  other 'Abort' errors should also be investigated.
+            pytest.skip("Investigate occurrence logic of 'TaskStateAborted' errors")
```

Comment:
Is this also not working against AWS or just localstack?

Reply:
It works against AWS. For LocalStack I first want to collect more insight into when it is triggered before updating the interpreter

Reply:
nit: You could in this case extend the if case by a check against `is_aws_cloud()` as well to make sure we only skip it when running against LS